packages/vuetify/src/components/VDatePicker/mixins/date-picker-table.ts

```diff
@@ -192,5 +193,13 @@ export default mixins(
         directives: [touchDirective],
       }, [transition])
     },
+    isInRange (value: string): boolean {
+      if (this.range && this.value.length === 2) {
+        const [from, to] = [...this.value as string[]].sort()
+        return from <= value && value <= to
```

Comment:
This is all down to personal preferences, but for me
```
value >= from && value <= to
```
is much clearer

Reply:
I prefer this way, it's like in mathematics from &leq; value &leq; to